### PR TITLE
[DO NOT MERGE] common/logging: send logs to stdout by default

### PR DESCRIPTION
**Pull request by ianvernon**
Opt-in to sending logs to syslog instead of setting up handler by
default. Output logs to stdout by default

Signed-off by: Ian Vernon <ian@covalent.io>

Fixes: #770

### CODE REVIEW

File: common/logging.go [hunk 1]
```diff
@@ -101,26 +100,23 @@ func setFireLevels(level logrus.Level) []logrus.Level {
...
-	if err != nil {
-		return err
-	}
-
 	// Logrus has a default logger that outputs to os.stderr. Set this
-	// default output to go to ioutil.Discard to not have duplicate logs.
-	logrus.SetOutput(ioutil.Discard)
-	setupSyslog(valuesToValidate, tag, debug)
+	// default output to go to os.Stdout to not have duplicate logs.
+	logrus.SetOutput(os.Stdout)
```
Comment (aanm) [inline common/logging.go:105]: I think this should be under `default:` and we have the problem solved

Comment (ianvernon) [reply common/logging.go:105]: The for loop will never be entered if there is no log-driver parameter passed into cilium. I prefer keeping 'default' for all non-supported loggers.

Comment (aanm) [reply common/logging.go:105]: oh it's inside a for loop. I think the `logrus.SetOutput(os.Stdout)` should be only set if `len(loggers) == 0`. Isn't what happens in the docker image?

**Comment by ianvernon** [comment]
Removing this branch / PR and putting up a new one with all fixes related to the logging framework change.

**Pull request closed without merging**